src/slash-bedrock/share/brl-fetch/distros/fedora

```diff
@@ -78,8 +108,14 @@ bootstrap_deps() {
 	echo "${package_manager} rpm fedora-release filesystem fedora-gpg-keys"
 }
 
-fetch() {
-	suffix="releases/${target_release}/Everything/${distro_arch}/os"
+fetch() {	
+	if [[ $(list_development | tr -d '[:space:]') == *${target_release}* ]]; then
+		url_type="development"
+	else
+		url_type="releases"
+	fi
```

Comment:
I see this conditional is called twice.  I'm glad you didn't make this a top level variable, as network calls would be surprising at that point.

My instinct would have been to make this a function to ensure we don't change it in one place and forget to change it in the other.  If you hadn't considered doing so, please do consider it.  If you have and disagree, that's fine; I'm not that picky about it and won't block merging over it.

Reply:
I can do that. I wasn't sure if you would prefer lots of functions (not doing very much) fewer functions but that did more, so I went with the one I would go with (an if else statement isn't very big)